### DISCUSSION TITLE
Server: ARK Survival of the Fittest

### DISCUSSION THREAD

**Discussion by X-Nightmare-X** [GameServerManagers/LinuxGSM#4130] [Ideas]
Hi, kann man den ARK Server für SotF einfach umbasteln oder das als Mod installieren?
Ansonsten wäre es mega, wenn man dafür einen extra Eintrag/Server erstellen könnte.
Bin mir gerade nicht ganz sicher wie ich das am besten mache.
Ansonsten ist LinuxGSM echt absolut genial und gut gemacht. Mein Respekt an das Tool.
Würde mich freuen, wenn mir das jemand genauer erklären kann, wie ich SotF installiere bzw. ob ich das als Mod mit in den normalen ARK Server packen kann und wie ich das dann einstellen und zum Laufen bringe.
Danke im Voraus

**Comment by Claiyc** [comment]
Es gibt eine dedizierte steam app für sotf: https://steamdb.info/app/445400/depots/
In dem Fall wäre es tatsächlich eine Überlegung wert, das als eigenständigen server in lgsm zu implementieren. Du kannst gerne [hier](https://github.com/GameServerManagers/LinuxGSM/issues/new?assignees=&labels=type%3A+game+server+request&template=server_request.yml&title=%5BServer+Request%5D%3A+) einen entsprechenden server request erstellen (auf englisch)
